packages/openid4vc/src/openid4vc-issuer/repository/OpenId4VcIssuanceSessionRecord.ts

```diff
@@ -40,6 +40,11 @@ export interface OpenId4VcIssuanceSessionAuthorization {
    */
   codeExpiresAt?: Date
 
+  /**
+   * Refresh token that can be used to refresh the access token.hat can be used to refresh the access token.
```

Comment:
```suggestion
   * Refresh token that can be used to refresh the access token.
```

Reply:
I don't think we should store the refresh token, but rather make it a signed jwt with the needed metadata. That way it's more flexible and we can also reuse it for refresh. 

We also don't store the access token.